management/distance-target.yaml

```diff
@@ -0,0 +1,261 @@
+              schema:
+                $ref: './schemas/distance-target.yaml#/schemas/ServiceUnavailable'
+        200:
+          description: The newly created Distance Target
+          content:
+            application/json:
+              schema:
+                $ref: './schemas/distance-target.yaml#/schemas/DistanceTarget'
+  '/v1/flight/{flightId}/distance/batch':
+    post:
+      tags:
+        - distance-targeting
+      operationId: batchUpload
+      security:
+        - ApiKeyAuth: []
+      parameters:
+        - name: flightId
+          description: The Flight Id to associate with the Distance Targeting geometry object
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int32
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                ReplaceExisting:
+                  type: boolean
+                  nullable: true
+                Geometries:
+                  type: array
+                  items: 
+                    $ref: './schemas/distance-target.yaml#/schemas/BatchUpload'
+      responses:
+        200:
+          description: The newly uploaded Distance Target(s)
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  InsertedGeometries: 
+                    type: integer
+                    format: int32
+  '/v1/flight/{flightId}/distance/{geometryId}':
+    put:
+      tags:
+        - distance-targeting
+      operationId: update
+      security:
+        - ApiKeyAuth: []
+      parameters:
+        - name: flightId
+          description: The Flight Id to associate with the Distance Targeting geometry object
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int32
+        - name: geometryId
+          description: The Id of the Distance Targeting geometry
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int32
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                Distance:
+                  type: integer
+                  format: float
+                  nullable: true
+                Latitude:
+                  type: integer
+                  format: float
+                  nullable: true
+                Longitude:
+                  type: integer
+                  format: float
+                  nullable: true
+                StreetAddress:
+                  type: string
+                  nullable: true
+      responses:
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: './schemas/distance-target.yaml#/schemas/BadRequest'
```

Comment:
Does a bad request really return an object result?

Reply:
Yeah I just tested it and I got:
{"Errors":
[{
  "message":"Address cannot be resolved",
  "values":{
    "Distance":42,
    "StreetAddress":"@#$%",
    "FlightId":19003355
  }
]}
}

Reply:
@calliebensel Can you check with the management team to make sure this structure is stable and we can rely on it? Also, the name should specify that it's a result type: `BadRequestResponse` or something similar.

Reply:
Updated